README.md

```diff
@@ -192,9 +198,10 @@ spec:
   source:
     repoURL: https://unikorn-cloud.github.io/helm-cluster-api
     chart: cluster-api-bootstrap-kubeadm
-    targetRevision: v0.1.9
+    targetRevision: v0.2.0
   destination:
-    server: https://172.18.255.200:443
+    server: ${TARGET_CLUSTER}
```

Comment:
Change to `https://kubernetes.default.svc`.  Most end users will expect to copy & paste.